server/server.go

```diff
@@ -174,31 +174,32 @@ func (s *Server) Read(ctx context.Context, req *openfgapb.ReadRequest) (*openfga
 	))
 	defer span.End()
 
-	modelID, err := s.resolveAuthorizationModelID(ctx, store, req.GetAuthorizationModelId())
-	if err != nil {
-		return nil, err
-	}
-	span.SetAttributes(attribute.KeyValue{Key: "authorization-model-id", Value: attribute.StringValue(modelID)})
-
 	q := commands.NewReadQuery(s.datastore, s.tracer, s.logger, s.encoder)
 	return q.Execute(ctx, &openfgapb.ReadRequest{
-		StoreId:              store,
-		TupleKey:             tk,
-		AuthorizationModelId: modelID,
-		PageSize:             req.GetPageSize(),
-		ContinuationToken:    req.GetContinuationToken(),
+		StoreId:           store,
+		TupleKey:          tk,
+		PageSize:          req.GetPageSize(),
+		ContinuationToken: req.GetContinuationToken(),
 	})
 }
 
+// ReadTuples returns all tuples for a given store.
+//
+// Deprecated: Please use Read with an empty tuple instead.
```

Comment:
> Please use Read without a request body instead.

Reply:
The request body may still include pagination options. So maybe
> Please use Read with a null tuple instead
??